pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -157,16 +157,12 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 
 	// Find the network object using the provided network name
```

Comment:
A couple comments are left over referring to finding by network name, which is not being done any more.